selfconference/selfconf-ios
# App icon update

## What does this PR do :question:
This PR includes updates to the app's AppIcon set. Mac and Apple Watch icon children were deleted, and unassigned icon children for iPhone and iPad were added.

No app functionality was changed.


## How to test it :microscope:
1. In XCode, note that the project builds with no warnings related to the AppIcon set, and all AppIcon children are assigned (see screenshot)
2. The actual app should still work as expected (see gif)


## Any background info you would like to include :white_check_mark:
The issue associated with this PR is: https://github.com/selfconference/selfconf-ios/issues/8


## Screenshots (if applicable) :camera:
![Screen Shot 2019-05-23 at 9 20 27 AM](https://user-images.githubusercontent.com/8409475/58256953-e7fb6480-7d3d-11e9-9b8d-3de2dcc063b6.png)

![selfconf-pr-app-icon-update](https://user-images.githubusercontent.com/8409475/58256961-ecc01880-7d3d-11e9-9747-29b736d542d1.gif)
